src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+  - /docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alert-quality-management-tutorial/
+  - /docs/new-relic-solutions/observability-maturity/aqm-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide
+---
+
+Teams suffer from alert fatigue when experiencing high alert volumes and alerts that aren't aligned to business impact. When this happens,incident responders often assume the alerts are false and have no business impact. In turn, they may start to prioritize easy-to-resolve alerts over others or close unresolved incidents to stay within their response time targets. This almost always results in slower incident response and increased scope and severity when major issues occur.
+
+Alert quality management (AQM) focuses on reducing the number of nuisance incidents so that you focus only on alerts with true business impact. This reduces alert fatigue and ensures that you and your team focus your attention on the right places at the right times.
+
+You're a good candidate for AQM if:
+
+* You have too many alerts.
+* You have alerts that stay open for long time periods.
+* You have a lot of alerts aren't relevant.
+* Your customers discover your issues before your monitoring tools do.
+
+<Callout variant="tip">
+Want to try a hands on learning approach before you start implementing this in your account? Check out the [alert quality management lab](https://learn.newrelic.com/hands-on-lab-alert-quality-management).
+</Callout>
+
+## Why use alert quality management? [#why-aqm]
+
+By using an alert strategy based on measuring business impact, you'll decrease response time and increase awareness of critical events. As you improve your alert signal to noise ratio, you'll reduce confusion and be able to rapidly identify and isolate the root cause of your problems. The overall goal of alert quality management is to lower the volume of less valuable alerts while providing easier ways to identify when more valuable incidents are created. This results in:
+
+* Increased uptime and availability.
+* Reduced mean time to resolution (MTTR).
+* Decreased alert volume.
+* The ability to easily identify alerts that are not valuable, so you can either make them valuable or remove them.
+
+AQM uses metrics to help you measure these, called **Key performance indicators**. For more on KPIs, see below.
+
+## Using Key performance indicators [#kpi]
+
+KPIs help you to find the noisiest and least valuable alerts so you can improve their value or remove them. You'll use the AQM process to collect and measure incident volume and engagement KPIs, and in turn, you'll use the long term metric trends to show real business impact to management and stakeholders. Below, you'll find information on all the KPIs, as well as a NRQL query for each one to help you monitor them from anywhere in the New Relic UI. 
```

Comment:
`you'll use the long term metric trends to show real business impact to management and stakeholders`

This feels a bit bloated and business-speak-y